lib/fileutils.rb

```diff
@@ -1642,6 +1642,12 @@ def install(src, dest, mode: nil, owner: nil, group: nil, preserve: nil,
       st = File.stat(s)
       unless File.exist?(d) and compare_file(s, d)
         remove_file d, true
+        begin
+          require 'pathname'
+        rescue LoadError
+          raise "cannot load pathname"
```

Comment:
Would a more descriptive error message like `raise  "Unable to create #{Pathname.new(d).parent}.  Cannot load module pathname."` be more helpful?

Reply:
That doesn't work, as the raise message requires pathname to be loaded...